.travis.yml

```diff
@@ -102,7 +97,7 @@ script:
     # capture the log, but only print it if `make deps` fails
     # try to show the end of the log first, because this log might be very long (> 4MB)
     # and thus be truncated by travis
-    - moreutils/mispipe "make $BUILDOPTS NO_GIT=1 -C deps 2> deps-err.log" "$BAR" > deps.log ||
+    - moreutils/mispipe "make \$BUILDOPTS NO_GIT=1 -C deps 2> deps-err.log" "$BAR" > deps.log ||
```

Comment:
Trying to figure out why mac wouldn't build this PR is why I started down the path of trying to see if the failure was caused by this mis-setting of BUILDOPTS.